packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -168,6 +169,10 @@ export async function createConsolidatedFromConversions({
     );
   }
 
+  ingestPatientIntoAnalyticsPlatform({ cxId, patientId }).catch(
+    processAsyncError("createConsolidatedFromConversions.ingestPatientIntoAnalyticsPlatform")
+  );
```

Comment:
Trigger analytics ingestion on consolidated bundle (re)creation.

Reply:
it has always been this, right, see the call to ingestConsolidatedIntoSearchEngine

we should def start emiting events instead, though